src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed');
```

Comment:
@whipermr5 while we wait for @damithc's input, what would be your opinion on the [above suggestion](https://github.com/TEAMMATES/teammates/pull/7811#discussion_r129743402) about defaulting to `false` if the value is undefined?